tests/ssg_test_suite/common.py

```diff
@@ -308,3 +325,34 @@ def iterate_over_rules():
                 directory=tests_dir, id=full_rule_id, short_id=short_rule_id,
                 files=scripts)
             yield result
+
+
+INSTALL_COMMANDS = dict(
+        fedora=("dnf", "install", "-y"),
+        rhel7=("yum", "install", "-y"),
+        rhel8=("yum", "install", "-y"),
+)
+
+
+def install_packages(domain_ip, platform, packages):
+    machine = "{0}@{1}".format(REMOTE_USER, domain_ip)
+    log_file_name = os.path.join(LogHelper.LOG_DIR, "env-preparation.log")
+
+    with open(log_file_name, 'a') as log_file:
+        print("Installing packages", file=log_file)
+        execute_remote_command(
+                machine, INSTALL_COMMANDS[platform] + tuple(packages), log_file,
+                "Couldn't install required packages {packages}".format(packages=packages))
+
+
+def benchmark_cpes_to_platform(cpes):
+    for cpe in cpes:
+        if "fedora" in cpe:
+            return "fedora"
+        if "redhat:enterprise_linux" in cpe:
+            version = re.match(r"enterprise_linux:(\d)+:", cpe).groups()[1]
```

Comment:
This causes problems for me when running rule test scenarios against a RHEL 8 VM target.

```suggestion
            version = re.search(r"enterprise_linux:(\d+)", cpe).groups()[0]
```

Reply:
It turned out to be even more complicated, but thanks for the suggestion - you were right on the search part.